src/components/BuySellForm/BuySellForm.tsx

```diff
@@ -87,6 +94,12 @@ const BuySellForm = ({ advertId, isModalOpen, onRequestClose }: TBuySellFormProp
 
     const { exchangeRate } = api.exchangeRates.useGet(local_currency);
 
+    const [hasCounterpartyRateChanged, setHasCounterpartyRateChanged] = useState(false);
+    const [hasRateChanged, setHasRateChanged] = useState(false);
+    const [inputValue, setInputValue] = useState(min_order_amount_limit_display ?? '0');
```

Comment:
minOrderAmountLimitDisplay

Reply:
since its destructured from the advertInfo as min_order_amount_limit_display i used the snake case value here, I think we can convert these values in another PR to camel case